pkg/core/gzip/gzip.go

```diff
@@ -22,14 +21,20 @@ import (
 // Gzip implements the gzip command.
 type Gzip struct {
 	core.Base
+	cmd     string
 	cmdLine *flag.FlagSet
 }
 
 // New returns a new Gzip command.
 func New() core.Command {
-	g := &Gzip{
-		cmdLine: flag.NewFlagSet("gzip", flag.ContinueOnError),
-	}
+	g := &Gzip{cmd: "gzip"}
+	g.Init()
+	return g
+}
+
+// NewGzcat returns a new Gzip command.
+func NewGzcat() core.Command {
```

Comment:
I still don't quite get this. What if the command using NewGzcat is not called gzcat? Then this cmd: is wrong. 

I'm sorta unsure about this different from exec.Command. It's as though exec.Command hardwire arg0 to be "sh" or something. 

Do we really do this in the other packages? How did I miss it?
Sorry, I'm a bit confused.